### PR TITLE
Bump to version 1.5.6

### PR DESCRIPTION
**Pull request by leilenah**
## Frontend

* Normalize worksheet item sort keys during get_worksheet_info. (#4168)

## CLI

None

## Backend

None

## Dev / docs / CI

None

### CODE REVIEW

**Comment by leilenah** [comment]
@wwwjn not sure that we should be updating `codalab/lib/upload_manager.py` in this PR cc @epicfaace

**Comment by wwwjn** [comment]
> codalab/lib/upload_manager.py

Should we only update the version number in "bump version" PR?

File: codalab/lib/upload_manager.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def write_fileobj(
             os.environ['AZURE_STORAGE_CONNECTION_STRING'] = bundle_conn_str
         try:
             bytes_uploaded = 0
-            CHUNK_SIZE = 16 * 1024
+            CHUNK_SIZE = 4 * 1024
```
Comment (epicfaace) [inline codalab/lib/upload_manager.py]: That's right, I think these changes look unintentional and should be reverted.

Comment (wwwjn) [reply codalab/lib/upload_manager.py]: Thanks Ashwin and Leilenah! I will revert this (it's for local testing purpose)

**Review by wwwjn — APPROVED** [review]

**Pull request merged**